modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/PodcastSyncProcess.kt

```diff
@@ -132,6 +132,7 @@ class PodcastSyncProcess(
                 cacheStats()
                     .andThen(downloadAndImportHomeFolder())
                     .andThen(downloadAndImportFilters())
+                    .andThen(rxCompletable { downloadAndImportBookmarks() })
```

Comment:
I would like to push towards sync using Coroutines rather than Rx, this is why I have used them where I can.